src/core/Akka/Util/MatchHandler/LambdaExpressionCompiler.cs

```diff
@@ -25,19 +25,6 @@ public Delegate Compile(LambdaExpression expression)
         {
             return expression.Compile();
         }
-
-#if !CORECLR
-        /// <summary>
-        /// TBD
-        /// </summary>
-        /// <param name="expression">TBD</param>
-        /// <param name="method"></param>
-        /// <returns>TBD</returns>
-        public void CompileToMethod(LambdaExpression expression, MethodBuilder method)
```

Comment:
Byterot, these codes were never compiled before the cleanup (since we moved to single `netstandard2.0` platform target)